config/settings.yml

```diff
@@ -133,6 +133,7 @@
     :refresh_interval: 15.minutes
   :openshift:
     :refresh_interval: 15.minutes
+    :skip_container_images: false
```

Comment:
I prefer to use positive-oriented option names.  This way, you don't get into the `unless !no_ping` type of code unreadability. `skip: false` is already a double-negative.

In amazon we used the option named `get_public_images`... See https://github.com/ManageIQ/manageiq-providers-amazon/blob/master/config/settings.yml#L32 .  I recommend `get_images: true` or something similar here.